spec/issues/issue465_spec.rb

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+require 'spec_helper'
+
+describe Bunny::Session do
+  let(:connection) do
+    c = Bunny.new(
+        user: 'bunny_gem', password: 'bunny_password',
+        vhost: 'bunny_testbed',
+        port: ENV.fetch('RABBITMQ_PORT', 5672)
+    )
+    c.start
+    c
+  end
+
+  context 'after the connection has been manually closed' do
+    before :each do
+      connection.close
+    end
+
+    after :each do
+      connection.close if connection.open?
+    end
+
+    describe '#create_channel' do
+      it 'should raise an exception' do
+        expect {
+          connection.create_channel
+        }.to raise_error(Bunny::ConnectionManuallyClosed)
+      end
+    end
+
+    describe '#start' do
+      it 'should be possible to reopen it' do
```

Comment:
I didn't plan on supporting this but if we do and it's actually a safe thing to do in practice, no objections from me.